### PR TITLE
doc: release notes: add release notes for Central SMP Client sample

### PR DESCRIPTION
**Pull request by maje-emb**
Added information about changing the sample name
from Central DFU SMP to Central SMP Client.

Relates to #5023.

Signed-off-by: Marcin Jeliński <marcin.jelinski@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/releases/release-notes-latest.rst [hunk 1]
```diff
@@ -40,7 +40,14 @@ nRF9160
...
 
-There are no entries for this section yet.
+The following changes are relevant for the nRF52 and nRF53 Series.
+
+Bluetooth Low Energy
+--------------------
+
+* Updated:
+
+  * :ref:`ble_samples` - Changed bluetooth  sample name Central DFU SMP to :ref:`Central SMP Client <bluetooth_central_dfu_smp>`.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-latest.rst:50]: I believe this should go under heading Applications and samples instead of nRF5.

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-latest.rst (hunk 1 above)
Comment (maje-emb) [reply doc/nrf/releases/release-notes-latest.rst]: Fixed.

Comment (peknis) [reply doc/nrf/releases/release-notes-latest.rst]: So sorry, Marcin. I was wrong. It was in the right place already, after all.

File: doc/nrf/releases/release-notes-latest.rst [hunk 2]
```diff
@@ -102,4 +102,11 @@ The following list summarizes the most important changes inherited from the upst
...
 
-There are no entries for this section yet.
+In addition to documentation related to the changes listed above, the following documentation has been updated:
+
+Applications and samples
+------------------------
+
+* Bluetooth:
+
+  * :ref:`ble_samples` - Changed bluetooth  sample name Central DFU SMP to :ref:`Central SMP Client <bluetooth_central_dfu_smp>`.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-latest.rst:112]:
```suggestion
  * :ref:`ble_samples` - Changed the Bluetooth sample Central DFU SMP name to :ref:`Central SMP Client <bluetooth_central_dfu_smp>`.
```

Comment (peknis) [reply doc/nrf/releases/release-notes-latest.rst:112]: And please move this back to the location you had originally.

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

File: doc/nrf/releases/release-notes-latest.rst (hunk 2 above)
Comment (maje-emb) [reply doc/nrf/releases/release-notes-latest.rst]: Done.

**Review by grochu — APPROVED** [review]

File: doc/nrf/releases/release-notes-latest.rst (hunk 1 above)
Comment (maje-emb) [reply doc/nrf/releases/release-notes-latest.rst]: Modified. It is as before.

**Pull request merged**